.stylelint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "stylelint-config-recommended",
```

Comment:
```suggestion
```

The standard config [already extends recommended](https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L4) so we can remove this line.